packages/@webex/plugin-cc/src/services/types.ts

```diff
@@ -69,6 +88,78 @@ export interface DesktopProfileResponse {
   idleCodes: string[];
 }
 
+/**
+ * Represents the request to a AgentLogin
+ *
+ * @public
+ */
+export interface AgentLoginRequest {
+  /**
+   * A dialNumber field contains the number to dial such as a route point or extension.
+   */
+
+  dialNumber?: string;
+
+  /**
+   * The unique ID representing a team of users.
+   */
+
+  teamId: string;
+
+  /**
+   * The loginOption field contains the type of login.
+   */
+
+  loginOption: LoginOption;
+}
+
+export interface UserStationLogin {
```

Comment:
Why are we creating 2 types here for the same purpose?

Reply:
as discussed, this is copied from agentx and for public facing we have to maintain good naming and minimal params